examples/profile.php

```diff
@@ -70,89 +69,412 @@
                             </td>
                         </tr>
                         <tr>
                             <th scope="row">Family Name</th>
                             <td><?php echo $familyName ? $familyName->getValue() : '' ?></td>
-                            <td><?php echo ProfileHelper::getAttributeSources($familyName) ?></td>
-                            <td><?php echo ProfileHelper::getAttributeVerifiers($familyName) ?></td>
+                            <td>
+                                <table class="table">
+                                    <thead class="thead-light">
+                                        <tr>
+                                            <th>S/V</th>
+                                            <th>Value</th>
+                                            <th>SubType</th>
+                                        </tr>
+                                    </thead>
+                                    <tbody>
+                                        <?php if ($familyName) : ?>
+                                            <?php foreach($familyName->getSources() as $source): ?>
+                                                <tr>
+                                                    <td>Source</td>
+                                                    <td><?php echo $source->getValue() ?></td>
+                                                    <td><?php echo $source->getSubType() ?></td>
+                                                </tr>
+                                            <?php endforeach; ?>
+                                            <?php foreach($familyName->getVerifiers() as $verifier): ?>
+                                                <tr>
+                                                    <td>Verifier</td>
+                                                    <td><?php echo $verifier->getValue() ?></td>
+                                                    <td><?php echo $verifier->getSubType() ?></td>
+                                                </tr>
+                                            <?php endforeach; ?>
+                                        <?php endif; ?>
+                                    </tbody>
+                                </table>
+                            </td>
                         </tr>
                         <tr>
                             <th scope="row">Full Name</th>
                             <td><?php echo $fullName ? $fullName->getValue() : '' ?></td>
-                            <td><?php echo ProfileHelper::getAttributeSources($fullName) ?></td>
-                            <td><?php echo ProfileHelper::getAttributeVerifiers($fullName) ?></td>
+                            <td>
+                                <table class="table">
+                                    <thead class="thead-light">
+                                        <tr>
+                                            <th>S/V</th>
```

Comment:
@qzagarese I assumed `S/V` was shorthand, and we'd put `Source/Verifier` in the final table, what do you think?

Reply:
I think S/V is just the header title the value itself is displaying as 'Source' or 'Verifier'.